src/charm.py

```diff
@@ -309,9 +320,15 @@ def _get_k8s_ingress(self):
         if self._owasp_modsecurity_crs:
             annotations["nginx.ingress.kubernetes.io/enable-modsecurity"] = "true"
             annotations["nginx.ingress.kubernetes.io/enable-owasp-modsecurity-crs"] = "true"
+
+            nginx_modsec_file = "/etc/nginx/owasp-modsecurity-crs/nginx-modsecurity.conf"
+            logging.info(repr(self._owasp_modsecurity_custom_rules))
```

Comment:
Should this log message include some information about what is being logged in addition the rules?

Reply:
My bad, this was not supposed to be there.